lib/jekyll/strapi/strapihttp.rb

```diff
@@ -0,0 +1,44 @@
+##
+# This is a helper method to authenticate during getting data from Strapi instance.
+require "json"
+
+def strapi_request(url)
+    uri = URI(url)
+    req = Net::HTTP::Get.new(uri)
```

Comment:
I wonder how difficult it would be to have `Down::NetHttps`

Reply:
All the media files are located here: `https://api.example.com/api/upload/files?`